tests/breed-hpprocurveN1178Test.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+require_once '../wwwroot/inc/deviceconfig.php';
+
+class hpprocurveN1178Test extends RTTestCase
+{
+
+	// hpprocurveN1178Read8021QConfig
+	/**
+	 * @dataProvider providerHP8021QConfigSamples
+	 */
+
+	public function testHP8021QConfParser ($test_data)
+	{
+	foreach ($test_data as $test => $test_array)
+		{
+		$input = $test_array['input'];
+		$expected = $test_array['results'];
+		$test_descr = $test ." " . $test_array['descr'];
+		$actual = hpprocurveN1178Read8021QConfig ($input);
+		if (! $this->assertEquals($expected, $actual))
+			print_r("\n\tOK! ----> HP 802.1Q Config Parser test " . $test_descr);
+		}
+	}
+
+
+	// hpprocurveN1178ReadInterfaceStatus
+	/**
+	 * @dataProvider providerHPShowIntBriefSamples
+	 */
+
+	public function testHPIntBriefConfParser ($test_data)
+	{
+	foreach ($test_data as $test => $test_array)
+		{
+		$input = $test_array['input'];
+		$expected = $test_array['results'];
+		$test_descr = $test ." " . $test_array['descr'];
+		$actual = hpprocurveN1178ReadInterfaceStatus ($input);
+		if (! $this->assertEquals($expected, $actual))
+			print_r("\n\tOK! ----> HP Show Interface Brief Parser test " . $test_descr);
```

Comment:
When the arguments are not equal, `assertEquals()` throws instead of returning, so it is likely the message is never printed. Hence the test description could be just a comment (in the PHP code or in the input text).